1688.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {number} n
+ * @return {number}
+ */
+var numberOfMatches = function (n) {
+  let match = 0;
+
+  while (n > 1) {
+    if (n % 2) {
```

Comment:
I always do like `n % 2 === 0` to check if the number is odd or even, but this expression is way better than that!
I got a good tip from your good code 👍